### ISSUE TITLE
Restrain mouse cursor option for full screen resolution.

### ISSUE THREAD

**Issue by ivanovmv** [colobot/colobot#715]
With dual monitor setup mouse cursor can move into secondary display while game in full screen mode. This and #604 slightly improve colobot experience while multitasking.

**Comment by krzys-h** [comment]
I personally prefer to be able to go to the second display whenever I want, but sure, I guess we could add that as a config option.
And you I guess you meant something other than #604, that issue doesn't seem related at all ;)

**Comment by ivanovmv** [comment]
Added correct issue that i actually mean #609 :)
Yeah this thing is kinda preference not an issue.
Here is few things:
1. Free mouse cursor make sense with border less(full screen) windowed mode, colobot runs on "true" fullscreen mode and when you interact with something outside colobot is minimized to taskbar. (Here #609 works well)
2. Camera control scheme utilized by colobot engine (RTS like when camera rotates around your target when cursor near border) goes nuts when you move mouse outside colobot and it feels disorienting especially when you accidentally move cursor too far into other display.

**Comment by krzys-h** [comment]
> Camera control scheme utilized by colobot engine (RTS like when camera rotates around your target when cursor near border) goes nuts when you move mouse outside colobot and it feels disorienting especially when you accidentally move cursor too far into other display.

It is also an issue if you play in a window as I usually do while developing - see #709
Could be also fixed by stopping camera rotation as soon as you move out of the game window.

**Comment by ivanovmv** [comment]
Yes, read #709 one before. I thought that right click camera would work on top of standard camera control (don't know why). But now i can relate it to my issue... if you disable default camera control and add right click one it solves everything. BTW "Center camera" hotkey can be handy in both cases.
